minio/commonconfig.py

```diff
@@ -83,6 +83,16 @@ def fromxml(cls: Type[A], element: ET.Element) -> A:
             obj[key] = value
         return obj
 
+    @classmethod
+    def object_from_user_tags_text(cls: Type[A], string: str | None) -> A:
```

Comment:
Remove this method and move the logic into https://github.com/minio/minio-py/pull/1381/files#diff-321a5d1087d9a70eb00a39c6bfe8b7295ff787da4efe97f7fe7fd16aed5c2df3R265